features/signals_railway_signals.yaml

```diff
@@ -2829,6 +2833,71 @@ features:
       - { tag: 'railway:signal:route', value: 'DE-ESO:zs2' }
       - { tag: 'railway:signal:route:form', value: 'light' }
 
+  - description: tram switch signal
+    country: DE
+    icon:
+      match: 'railway:signal:switch:states'
+      cases:
+        - { exact: 'DE-BOStrab:w14', value: 'de/bostrab/w14', description: 'no trailing-point movement' }
+        - { exact: 'DE-BOStrab:w13', value: 'de/bostrab/w13', description: 'switch set to the left' }
+        - { exact: 'DE-BOStrab:w3', value: 'de/bostrab/w3', description: 'switch set to the left (low speed)' }
+        - { exact: 'DE-BOStrab:w12', value: 'de/bostrab/w12', description: 'switch set to the right' }
+        - { exact: 'DE-BOStrab:w2', value: 'de/bostrab/w2', description: 'switch set to the right (low speed)' }
+        - { exact: 'DE-BOStrab:w11', value: 'de/bostrab/w11', description: 'switch set to straight-ahead' }
+        - { exact: 'DE-BOStrab:w1', value: 'de/bostrab/w1', description: 'switch set to straight-ahead (low speed)' }
+        - { exact: 'DE-BOStrab:w0', value: 'de/bostrab/w0', description: 'switch in use' }
+      default: 'de/bostrab/w-unknown'
+    tags:
+      - { tag: 'railway:signal:switch', value: 'DE-BOStrab:w' }
+      - { tag: 'railway:signal:main:form', value: 'light' }
+
+  - description: tram no trailing-point movement W 14 (sign)
+    country: DE
+    icon: { default: 'de/bostrab/w14-sign' }
+    tags:
+      - { tag: 'railway:signal:switch', value: 'DE-BOStrab:w' }
+      - { tag: 'railway:signal:switch:form', value: 'sign' }
+      - { tag: 'railway:signal:switch:states', exact: 'DE-BOStrab:w14' }
```

Comment:
This one has to go before the general tram switch signal, because otherwise the tags will match first on the more general signal (it will also match `DE-BOStrab:w` with form `light`)